src/test/scala/io/iohk/ethereum/network/p2p/PeerActorSpec.scala

```diff
@@ -401,7 +401,16 @@ class PeerActorSpec extends FlatSpec with Matchers {
       extraData = ByteString("0"),
       mixHash = ByteString("0"),
       nonce = ByteString("0"))
-    blockchain.save(testGenesisHeader)
+
+    val testGenesisBlockBody: BlockBody = BlockBody(
+      transactionList = Seq[SignedTransaction](
+      ),
+      uncleNodesList = Seq[BlockHeader](
+      ))
```

Comment:
Minor: `BlockBody(Nil, Nil)` also does the job and looks much simpler

Reply:
fair point